com.stansassets.google-doc-connector-pro/Runtime/Core/GoogleDocConnectorLocalization.cs

```diff
@@ -24,5 +29,17 @@ internal static void SpreadsheetIdSet(string newSpreadsheetId)
             LocalizationClient.ClearDefault();
             SpreadsheetIdChanged.Invoke();
         }
+
+        /// <summary>
+        ///  Set a new spreadsheet ID and sheet ID to be used for the Localization client
+        /// </summary>
+        /// <param name="newSpreadsheetId"></param>
+        /// <param name="newSheetId"></param>
+        internal static void SpreadsheetIdSet(string newSpreadsheetId, int newSheetId)
```

Comment:
I think we can have only one `SpreadsheetIdSet` method with an optional parameter instead of two (like it is right now).
Also, the ideal name for this method would be `internal static void SetSpreadsheet(string spreadsheetId, int sheetId = 0)`